pages/tools.jsx

```diff
@@ -52,8 +54,9 @@ var ToolsColumn = React.createClass({
           src1x={this.props.src1x}
           src2x={this.props.src2x}
           alt=""
-          className="vertical-layout">
-          <h2><a href={this.props.link}>{this.props.name}</a></h2>
+          className="vertical-layout"
+          externalLink>
```

Comment:
can we keep with "assigned value" here and say `externalLink={true}` or the like? As a plain string followed by `>` this looks like an unmatched component tag (to human eyes. JSX converter doesn't care of course =D)